CentralManagement/ChocoCCM Command Reference/Remove-CCMDeploymentStep.md

```diff
@@ -0,0 +1,106 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Remove-CCMDeploymentStep
+
+## SYNOPSIS
+Removes a deployment plan
+
+## SYNTAX
+
+```
+Remove-CCMDeploymentStep [-Deployment] <String> [-Step] <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Removes the Deployment Plan selected from a Central Management installation
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Remove-CCMDeploymentStep -Name 'Super Complex Deployment' -Step 'Kill web services'
+```
+
+### EXAMPLE 2
+```
+Remove-CCMDeploymentStep -Name 'Deployment Alpha' -Step 'Copy Files' -Confirm:$false
+```

```

Comment:
Examples need descriptions of what happens when you run them. This probably needs going over for a lot of these docs.